website/docs/components/side-panel/examples/access_to_internal_components.tsx

```diff
@@ -11,7 +11,7 @@ const Demo = () => {
       <Button onClick={() => setVisible(true)}>Show SidePanel</Button>
       <SidePanel closable={false} visible={visible} onClose={() => setVisible(false)}>
         <SidePanel.Overlay>
-          <SidePanel.Panel mt={20}>
+          <SidePanel.Panel aria-label='Blah-blah-blah'>
```

Comment:
maybe some another label?

Reply:
I guess as content is meaningless, meaningless aria-label is ok :)

Reply:
ahaha I think it would be better to put meaningful content and label here, I can do it